src/crypto/verification/request/ToDeviceChannel.js

```diff
@@ -179,7 +179,9 @@ export class ToDeviceChannel {
         const isAcceptingEvent = type === START_TYPE || type === READY_TYPE;
         // the request has picked a ready or start event, tell the other devices about it
         if (isAcceptingEvent && !wasStarted && isStarted && this._deviceId) {
-            const nonChosenDevices = this._devices.filter(d => d !== this._deviceId);
+            const nonChosenDevices = this._devices.filter(
+                d => d !== this._deviceId && d !== this._client.getDeviceId(),
```

Comment:
Hmm, how was this manifesting...? A bit surprised this wasn't more a of wide spread problem...

Reply:
on accepting a self-verification, the verifier sent the cancellation for all devices other than the verifiee.  Which included the verifier, who saw the cancellation and promptly cancelled the verification.  I've seen it in the wild a few times before.  The workaround is to manually ignore cancellations to yourself whilst you're self-verifying, but frankly I think it's safer to do it in here.